offline-environment.md

```diff
@@ -0,0 +1,63 @@
+# Installing OpenFisca in an offline environment
+
+Some people need to install OpenFisca on a server which has no Internet access.
+Here is a recipe explaining how to do.
```

Comment:
“If you need to install OpenFisca on a server with no Internet access, here is how to do it.”